x-pack/plugins/cases/server/saved_object_types/cases.ts

```diff
@@ -177,7 +177,7 @@ export const createCaseSavedObjectType = (
         },
       },
       severity: {
-        type: 'keyword',
+        type: 'short',
```

Comment:
FYI, this will likely be the last Kibana version where mutating existing field types is allowed

Reply:
Thanks, @pgayvallet! We are shipping this now with that in mind 😄 There is one more similar PR coming.